generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.generator.operations;
+
+import org.corfudb.generator.State;
+import org.corfudb.runtime.MultiCheckpointWriter;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Created by maithem on 7/14/17.
+ */
+@Slf4j
+public class CheckpointOperation extends Operation {
+
+    public CheckpointOperation(State state) {
+        super(state);
+    }
+
+    @Override
+    public void execute() {
+        try {
+            MultiCheckpointWriter mcw = new MultiCheckpointWriter();
+            mcw.addAllMaps(state.getMaps());
+            long trimAddress = mcw.appendCheckpoints(state.getRuntime(), "Maithem");
+            state.setTrimMark(trimAddress);
+            Thread.sleep(1000 * 30 * 1);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)